daemon/container.c

```diff
@@ -293,6 +299,13 @@ container_new_internal(const uuid_t *uuid, const char *name, container_type_t ty
 
 	container->images_dir = mem_strdup(images_dir);
 
+	/* create image_dir already so we can store a pseudo file as a
+	 * marker that the token aossicated with the container has been initialized
+	 */
+	if (mkdir(images_dir, 0755) < 0 && errno != EEXIST) {
+		ERROR_ERRNO("Cound not mkdir container directory %s", images_dir);
```

Comment:
i dont't think that would be benefitial. the respective container will not be able to start up anyway

Reply:
we should abort container initialization as soon as it becomes clear there is an issue
Therefore, please add goto error;

Also, it would be cleaner move the directory creation below the assignments

Reply:
added goto;

I believe moving the creation of the directory away from where its pathh is set would not adhere to the style of the rest of the function